plugins/socks5-proxy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: socks5-proxy
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/yokawasa/kubectl-plugin-socks5-proxy
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/yokawasa/kubectl-plugin-socks5-proxy/archive/v1.0.0.zip
+    sha256: b8d96ab7b7d2ba23d973eeb3a22080829073ae79f8cad5966740fc1ceaf766d3
+    files:
+    - from: "./*/kubectl-socks5-proxy"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    bin: "./kubectl-socks5-proxy"
+  shortDescription: A kubectl plugin that creates a local SOCKS5 proxy through which you can access to Services or Pods in a Kubernetes cluster.
+  description: |
+    A kubectl plugin that creates a local SOCKS5 proxy through which you can
+    access to Services or Pods in a Kubernetes cluster.
+    
+    What the plugin actually does is that it create a SOCKS proxy server Pod 
```

Comment:
If you are using a widely known standard container as the proxy server, maybe mention this in parentheses? For example: "..creates a SOCKS5 proxy server Pod (based on alpine).."

Reply:
I added base container name `.. create a SOCKS proxy server Pod (based on serjs/go-socks5-proxy)`